src/api/dexPriceEstimator/DexPriceEstimatorApi.ts

```diff
@@ -109,6 +129,23 @@ export class DexPriceEstimatorApiImpl implements DexPriceEstimatorApi {
     return `${baseUrl}markets/${baseTokenId}-${quoteTokenId}?atoms=true&hops=${hops}`
   }
 
+  public async getOrderBookData(params: OrderBookParams): Promise<RawApiData> {
+    try {
+      const url = await this.getOrderBookUrl(params)
+
+      const res = await fetch(url)
+      return await res.json()
```

Comment:
why return `await`?

Reply:
so that it triggers `catch` here
otherwise we would return a potentially rejected promise